src/client/app/core/datepicker/datepicker.js

```diff
@@ -0,0 +1,78 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('app.core')
+        .controller('datePickerCtrl', datePickerCtrl)
```

Comment:
I'd recommend using consistent naming.
Change this to DatePickerController because this is something you use in other controllers.